test/test_models/models/simple_fc_backbone.ini

```diff
@@ -0,0 +1,3 @@
+[]
```

Comment:
how does iniparser handle this? section with empty name?

Reply:
Yes, for now it is just an empty name

Reply:
cool.